src/tasks/collectors.ts

```diff
@@ -28,5 +28,15 @@ bot.tasks.set(`collectors`, {
       // Reject the promise so code can continue in commands.
       return collector.reject();
     });
+
+    bot.buttonCollectors.forEach((collector, key) => {
+      // This collector has not finished yet.
+      if (collector.createdAt + collector.duration > now) return;
+
+      // Remove the collector
+      bot.buttonCollectors.delete(key);
+      // Reject the promise so code can continue in commands.
+      return collector.reject();
```

Comment:
can u add reasons in all the rejects here plz

collector.reject("The button was not pressed in time."); something like this

Reply:
Sure, I assume this should be similar with other collectors in this file? Should I add similar reasons there too?